tests/fromRdf/0027-out.jsonld

```diff
@@ -0,0 +1,60 @@
+{
+  "@graph": [
+    {
+      "@id": "http://example.com/boolean-native",
+      "http://example.com/example": [
+        {
+          "@type": "http://www.w3.org/2001/XMLSchema#boolean",
+          "@value": true
+        },
+        {
+          "@type": "http://www.w3.org/2001/XMLSchema#boolean",
+          "@value": false
+        }
+      ]
+    },
+    {
+      "@id": "http://example.com/boolean-object",
+      "http://example.com/example": [
+        {
+          "@type": "http://www.w3.org/2001/XMLSchema#boolean",
+          "@value": "True"
+        },
+        {
+          "@type": "http://www.w3.org/2001/XMLSchema#boolean",
+          "@value": "False"
+        }
+      ]
+    },
+    {
+      "@id": "http://example.com/number-native",
+      "http://example.com/example": [
+        {
+          "@type": "http://www.w3.org/2001/XMLSchema#integer",
+          "@value": 1
+        },
+        {
+          "@type": "http://www.w3.org/2001/XMLSchema#decimal",
+          "@value": 1.1
```

Comment:
This causes issues, because it round-trips to `"1.1E0"^^xsd:decimal`, which is ill-formed (xsd:decimal does not support scientific notation in its lexical space)